src/Aquifer.API/Endpoints/Bibles/Book/List/Endpoint.cs

```diff
@@ -23,7 +23,10 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
                 Code = bbc.Book.Code,
                 Number = (int)bbc.BookId,
                 LocalizedName = bbc.DisplayName,
-                TotalChapters = bbc.Book.Chapters.Count,
+                TotalChapters = bbc.ChapterCount,
+
+                // TODO TotalVerses here should come from actual Bible data, not the standard BookChapters English versification data.
```

Comment:
Before we do this we need to look at the Bible Well and confirm that it can handle getting back different `TotalVerses` data for different `bibleId`s.